ores/api.py

```diff
@@ -107,10 +122,9 @@ def _score_request(self, context, rev_ids, models):
         logging.debug("Sending score request for {0} revisions"
                       .format(len(rev_ids)))
         start = time.time()
-        response = requests.get(url, params=params,
-                                headers=self.headers,
-                                verify=True)
-
+        response = self._session.get(url, params=params,
```

Comment:
This is where an exception should be turned into JSON, probably with a `try` block to catch API errors.

Reply:
I think we should let errors that occur here bubble up.  I don't see much value in re-wrapping the error and we probably don't want to blindly handle a connection error or something like that.